docker/README.md

```diff
@@ -61,19 +61,19 @@ Start `btcd`, and then create an address for `Alice` that we'll directly mine
 bitcoin into.
 ```bash
 # Init bitcoin network env variable:
-$ export NETWORK="simnet" 
+$ export NETWORK="simnet"
 
 # Run the "Alice" container and log into it:
-$ docker-compose run -d --name alice lnd_btc
+$ docker-compose run -d --name alice alice
```

Comment:
I think it's a good idea to simplify the docs here, although it would be better to do something like:
`docker-compose exex -it --name alice` and name the container when it's run.